src/ui/components/userDetail/userRoles/UserRolesList.tsx

```diff
@@ -26,15 +25,15 @@ import DeleteUserRoleDialog from "../../userroles/components/dialogs/DeleteUserR
 import "./userRolesList.scss";
 
 type UserRolesListProps = {
-	roles: string[];
+	roles: string[] | undefined;
```

Comment:
Don't pass undefined to this component since it can't be shown if the content is no loaded.

Reply:
done